src/nrnoc/init.cpp

```diff
@@ -157,7 +157,7 @@ extern "C" void hoc_reg_watch_allocate(int type, NrnWatchAllocateFunc_t waf) {
 }
 
 // also for read
-typedef int (*bbcore_write_t)(void*, int, int*, double*, Datum*, Datum*, NrnThread*);
+typedef void (*bbcore_write_t)(double*, int*, int*, int*, double*, Datum*, Datum*, NrnThread*);
```

Comment:
```suggestion
using bcore_write_t = void (*)(double*, int*, int*, int*, double*, Datum*, Datum*, NrnThread*);
```

Reply:
Done in 2c8b3735124cbb19cd1ba5442e9c1eb1f64a4a9f :+1: